.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.pythonPath": "/Users/gfvante/coding/pytest-flask/venv/bin/python3.7",
```

Comment:
This doesn't look like it belongs here, it contains a local path in your machine. I suppose this was committed incorrectly?

Reply:
Oops.